src/components/Select/GroupHeading.tsx

```diff
@@ -0,0 +1,36 @@
+import clsx from 'clsx';
+import * as React from 'react';
+import { makeStyles } from '../../styles';
+import { GetClasses } from '../../typeUtils';
+
+export const GroupHeadingStylesKey = 'ChromaSelectGroupHeading';
+
+export const useStyles = makeStyles(
+  (theme) => ({
+    root: {
```

Comment:
Styling is not exactly my strong suit... I tried to follow some examples in the code but open to any feedback on how to make this better.

Reply:
Sure thing! I'll take a look this evening and can provide any feedback. We may defer to Casey as well for additional polish post-merge!

Reply:
Just chatted with @dexterca, sounds like he may have some time tomorrow to style this out a bit!